ldaptor/test/test_pureldap.py

```diff
@@ -28,14 +26,16 @@ def s(*l):
     """Join all members of list to a string. Integer members are chr()ed"""
     r=''
     for e in l:
-        if isinstance(e, types.IntType):
-            e=chr(e)
-        r=r+str(e)
+        if isinstance(e, int):
+            e = chr(e)
+        r = r + str(e)
     return r
 
+
 def l(s):
     """Split a string to ord's of chars."""
-    return map(lambda x: ord(x), s)
+    return [six.byte2int([x]) for x in s]
```

Comment:
Another `six.byte2int()` invocation taking a list when the plain old byte string should work.  I am pretty sure I saw at least one place in the code where this function was being used with a plain old byte string.

Reply:
I did a search, but I did not found any six.byte2int used without a list.